var/spack/repos/builtin/packages/cool/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Cool(CMakePackage):
+    """COOL provides specific software components and tools for the handling of the time
+       variation and versioning of the experiment conditions data."""
+
+    homepage = "https://coral-cool.docs.cern.ch/"
+    git      = "https://gitlab.cern.ch/lcgcool/cool.git"
+
+    tags = ['hep']
+
+    version('3.3.5', tag='COOL_3_3_5')
+    version('3.3.4', tag='COOL_3_3_4')
+    version('3.3.3', tag='COOL_3_3_3')
+
+    patch('cool.patch', level=0)
```

Comment:
Can you add a comment on top of the directive to explain provenance and reason of the patch?

Reply:
Done